llvm/unittests/IR/ConstantRangeTest.cpp

```diff
@@ -2788,4 +2788,31 @@ TEST_F(ConstantRangeTest, isSizeLargerThan) {
   EXPECT_FALSE(One.isSizeLargerThan(1));
 }
 
+TEST_F(ConstantRangeTest, MakeMaskNotEqualRangeExhaustive) {
```

Comment:
I'd suggest to also add one or two tests with explicit ranges (as we're not testing optimality).